### ISSUE TITLE
Support for EulerAncestralDiscreteScheduler

### ISSUE THREAD

**Issue by anonymous-atom** [kvablack/ddpo-pytorch#28]
Can DDPO be used with EulerAncestralDiscreteScheduler as a scheduler for finetuning the Diffusion Model ?

**Comment by anonymous-atom** [comment]
Can EulerAncestralDiscreteScheduler be patched to work with DDPO ?

**Comment by anonymous-atom** [comment]
@kvablack any thoughts on this ?
